react/Checkbox/Checkbox.less

```diff
@@ -40,16 +40,16 @@
     .focus();
   }
 
-  .input:checked + .label & {
-    background-color: @sk-mid-gray-light;
+  .label & {
+    background-color: @sk-white;
   }
 }
 
 .checkMark {
   display: block;
   width: 100%;
   height: 100%;
-  fill: @sk-black;
+  fill: @sk-focus;
```

Comment:
Can probably leave this as-is for now, but this will probably need to trigger a design discussion about whether our focus colours should be coupled to the form element design in this way, cc @stephaniemoss